since musl commit 0b80a7b0404b6e49b0b724e3e3fe0ed5af3b08ef lynx crashes when
resized (with netbsd curses).
the reason is that when ncurses isn't detected, lynx uses a hack when it
receives SIGWINCH: stop_curses(); start_curses();
it's not safe to call any stdio func from a signal handler, but that's what
happens here.
arguably netbsd-curses shouldnt call setvbuf() at all, as it is only valid
as the first operation on a stream. will be fixed separately.

--- lynx2.8.9dev.16/src/LYMainLoop.c
+++ lynx2.8.9dev.16.patched/src/LYMainLoop.c
@@ -6367,7 +6367,7 @@
 	     * are already filled based on the old size.  So we notify the
 	     * ncurses library directly here.  - kw
 	     */
-#if defined(NCURSES) && defined(HAVE_RESIZETERM) && defined(HAVE_WRESIZE)
+#if defined(HAVE_RESIZETERM) && defined(HAVE_WRESIZE)
 	    resizeterm(LYlines, LYcols);
 	    wresize(LYwin, LYlines, LYcols);
 #else
